### ISSUE TITLE
Meeting Discussion Point 6: Interacting with Data

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#393]
This post is being written until this line is removed.
Background to Meeting on May 24th 2016  (#386).

**Question**: The OSM consortium uses a number of different tools to disseminate and interact with project data - are these all effective?
**Question**: What new methods of collaborating on project data could be tried that would be accessible to the widest number of OSM participants?

### Resources:
OSM [Series 4 summary wiki](http://openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_%28TP%29_Series)
Lab books: [Labtrove](http://malaria.ourexperiment.org/?allblogs), Labarchives ([Ed Tse](https://au-mynotebook.labarchives.com/share/Edwin%2520Tse/MTYzLjh8MTUwOS8xMjYvVHJlZU5vZGUvMjUxNDEwNDEwOHw0MTUuOA==), [Chase Smith](https://mynotebook.labarchives.com/share/MCPHS%2520MedChem/MzUxLjB8MzY3Ny8yNzAvVHJlZU5vZGUvNDA0ODg3Njc4OHw4OTEuMA==)). See the Biological Evaluation ELN in particular for this Issue.
Compound Database: [All OSM Compounds in Google Sheet](http://tinyurl.com/OSM-Compounds), [interactive view on Cheminfo](http://www.cheminfo.org/flavor/malaria/Display_data.html), [direct download of Series 4 SDF](http://malaria.ourexperiment.org/data/files/17162/malaria-2016-05-18.sdf).

_Comments/suggestions below are welcome, as well as (briefly please) during the main webinar meeting on the 24th and its follow-up on June 8th._

## Relevant background:

**Comment by drc007** [comment]
A quick comment. The Google doc spreadsheet is not a database, it is a simple spreadsheet. The big advantage is that it was trivial to set up and populate. There are of course several potential advantages to having a proper online, chemically intelligent database. If anyone would like to set one up that would be great, perhaps an honours student project? Would be a publicly accessible entry on your CV ;-)

**Comment by cdsouthan** [comment]
As mentioned before, no single route of chemical structure and data surfacing has the speed and global connectivity of PubChem BioAssay. Submissions to ChEMBL that eventually surface in PubChem are good but have disadvantages.  So far, only 125/240 of the OSM structure list have exact matches (https://cdsouthan.blogspot.se/2015/05/entity-resolution-for-antimalarial.html). Some of 115 missing structures are either being synthesised or waiting for data. However, there is a discoverability argument for having all structures in PubChem, even the virtual designs (but tagged as such).

**Comment by mattodd** [comment]
Yes, to my mind the Google Doc is an excellent primary source of data. Easy to input, easy to edit and is backed up. I think we're all making the assumption that the data can be ported into any number of other places, such as a proper database. But of course we all want there to be a single place where we enter data.
@cdsouthan yes, I know you've made this good point before, and it remains a good point. My issue is just the word "submission", implying a manual process. I'm **sure** you've answered this before, but is there a way we can auto-submit from the Google Doc (or something derived automatically from the Google Doc that has the right format) into Pubchem or Chembl? @madgpap That would be powerful.

**Comment by cdsouthan** [comment]
OK, let me know when first-activity results go into the GD sheet for newly synthesised Series 4 strucs that are PubChem -ve.  I will then try to get to check out the improved assay submission system. This will allow us to asses auto-parsing (more or less Excel reformating) feasibility for piping direct to PubChem BioAssay
